source/_patterns/02-molecules/example-widget/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Example JS widget
+ *
+ * An example of a testable, standalone, javascript-driven widget that makes an api call, writes
+ * HTML, and does basic logic. This is a pretty naive re-render-on-data-change approach, using
+ * poor-man's Redux and jQuery
```

Comment:
I think this docblock could be expanded a little bit to explain future usages. I'm imagining the questions you'll be getting being best answered here. Specifically for this example widget it may be good to differentiate what we're doing in `src/index.js` vs `index.js`

Reply:
@jryanconklin are you looking for explanation around the idea of a source and an implementation?

Reply:
Yes indeed, I think even just saying "source and implementation" here would help as an example. Further since this will be the only example of this kind of widget, it might be good to list what the minimum requirements would be to make it work so it's replicable without people sending you basic questions.

Reply:
@jryanconklin better docs here and at `example-widget/src/index.js`.